bin/run_qc.py

```diff
@@ -177,5 +183,37 @@ def detect_bad_cycle_tile_in_run(dataset, args):
             print('Bad tiles are: ' + ', '.join([str(c) for c in bad_tiles[lane]]))
 
 
+def get_all_project_gvcfs(project_folder):
+        gvcfs = []
+        for path in os.walk(project_folder):
+            gvcf = [i for i in path[2] if i.endswith('.g.vcf.gz')]
+            if gvcf:
+                gvcfs.append(os.path.join(path[0], ''.join(gvcf)))
+        return gvcfs
+
+def peddy(dataset, args):
+    all_gvcfs = []
+    sample_ids = []
+    if args.samples:
+        cfg.merge(cfg['sample'])
+        for sample_id in args.samples:
+            sample_ids.append(sample_id)
+            s = get_document(sample_id)
+            project_id = s.get('project_id')
+            gvcf_file = s.get('user_sample_id') + '.g.vcf.gz'
+            gvcf = util.find_file(cfg['input_dir'], project_id, sample_id, gvcf_file)
+            all_gvcfs.append(gvcf)
+    elif args.projects:
+        cfg.merge(cfg['project'])
+        for project_id in args.projects:
+            samples_for_project = clarity.get_sample_names_from_project(project_id)
+            sample_ids.extend(samples_for_project)
+            project_folder = util.find_file(cfg['input_dir'], project_id)
+            all_gvcfs.extend(get_all_project_gvcfs(project_folder))
+
+    qc.Genotype_gVCFs(all_gvcfs)
+    p = qc.Peddy(dataset=dataset, ids=sample_ids)
+    p.run()
```

Comment:
As it stands, Peddy will try to run before the merged gvcf has been generated and crash. You could pass the Genotype_gVCFs object in via `previous_stages`, but would then need to call `luigi.build` as in `pipelines.__init__`. Alternatively, you can run each task manually:

```python
g = qc.Genotype_gVCFs(dataset=dataset, GVCFs=all_gvcfs, reference=args.reference)
g.run()
p = qc.Peddy(dataset=dataset, ids=sample_ids)
p.run()
```